NEWS.md

```diff
@@ -25,6 +25,9 @@ Language changes
 Compiler/Runtime improvements
 -----------------------------
 
+* The package precompilation process now saves native code into a "pkgimage", meaning that code generated during the
+  precompilation process will not require compilation after package load, resulting in much faster time to first
+  execution (TTFX), often referred to as time to first plot ([#44527]) ([#47184]).
```

Comment:
Would it be better to start with the consequence, and then describe how it was done? I.e., "Time to first execution (TTFX, sometimes called time to first plot) is greatly reduced. Package precompilation ..."